packages/frontend/src/components/UserSettings/UserAttributesPanel/UserAttributeModal.tsx

```diff
@@ -0,0 +1,201 @@
+
+                        {form.values.users?.length > 0 && (
+                            <Group>
+                                <Text w={200} fw={500}>
+                                    User email
+                                </Text>
+                                <Text fw={500}>Value</Text>
+                            </Group>
+                        )}
+
+                        {form.values.users?.map((user, index) => {
+                            return (
+                                <Group key={index}>
+                                    <Select
+                                        w={200}
+                                        name={`users.${index}.userUuid`}
+                                        placeholder="E.g. test@lightdash.com"
+                                        required
+                                        searchable
+                                        {...form.getInputProps(
+                                            `users.${index}.userUuid`,
+                                        )}
+                                        data={
+                                            orgUsers?.map((orgUser) => ({
+                                                value: orgUser.userUuid,
+                                                label: orgUser.email,
+                                            })) || []
+                                        }
+                                    />
+
+                                    <TextInput
+                                        name={`users.${index}.value`}
+                                        placeholder="E.g. US"
+                                        required
+                                        {...form.getInputProps(
+                                            `users.${index}.value`,
+                                        )}
+                                    />
+                                    <Button
+                                        pr={5}
+                                        leftIcon={
+                                            <MantineIcon icon={IconCircleX} />
+                                        }
+                                        color="red"
+                                        variant="outline"
+                                        onClick={() => {
+                                            form.setFieldValue(
+                                                'users',
+                                                form.values.users.filter(
+                                                    (_, i) => i !== index,
+                                                ),
+                                            );
+                                        }}
+                                    />
+                                </Group>
+                            );
+                        })}
+                        <Button
+                            w={200}
```

Comment:
(note for self - check this in preview env)

Reply:
you can ignore aligning it perfectly to first input and just make its weight "auto" (by removing the prop)

Reply:
This property is not set to allign all the inputs, but to alling the input with the headers, otherwise  `value` is off

![image](https://github.com/lightdash/lightdash/assets/1983672/e8d8b449-2536-457c-84b1-8cd6f398aba4)

Reply:
I think keeping User email and value labels as a separate line is not going to be consistent with the layout that im suggesting.
here's an idea - what if you set mantine input label prop only for the first row (where index===0)?
in that case label will have a proper spacing and will follow the inputs properly.